src/fsharp/TastReflect.fs

```diff
@@ -988,9 +988,10 @@ and ReflectAssembly(g, ccu: CcuThunk, location:string) as asm =
                 | None -> failwith (sprintf "TODO: didn't get back to CCU being compiled for local tcref %s" tcref.DisplayName)
             let reflAssem = ccuofTyconRef.ReflectAssembly :?> ReflectAssembly
             let tcrefR = reflAssem.TxTypeDef None tcref
-            match tinst with 
-            | [] -> tcrefR 
-            | args -> tcrefR.MakeGenericType(Array.map asm.TxTType (Array.ofList args))  
+            tcrefR
+            //match tinst with 
```

Comment:
I suspected that. The difficulty arose when I was trying to implement the Typar case https://github.com/colinbull/visualfsharp/blob/98aa10071d5c07cb3756f9a4e57f5f171f8ad136/src/fsharp/TastReflect.fs#L1009 my problem was I didn't know how to get any type information about the given type parameter. So given a MyRecord<string> all I was seeing was the Typar representing the `'a` and I didn't know how to resolve this to a type `string`. Hence this implementation which kinda skips the problem. I knew this was wrong but I think I'm missing some understanding of how things like this resolve.